Documentation/operations/system_requirements.rst

```diff
@@ -262,35 +249,32 @@ to change the packet scheduling algorithm.
 
         CONFIG_NET_SCH_FQ=m
 
+.. _advanced_features:
 
 Required Kernel Versions for Advanced Features
 ==============================================
 
-Cilium requires Linux kernel 4.9.17 or higher; however, development on
-additional kernel features continues to progress in the Linux community. Some
+Additional kernel features continues to progress in the Linux community. Some
 of Cilium's features are dependent on newer kernel versions and are thus
 enabled by upgrading to more recent kernel versions as detailed below.
 
-====================================================== 	===============================
-Cilium Feature                                          Minimum Kernel Version
-====================================================== 	===============================
-:ref:`concepts_fragmentation`				>= 4.10
-:ref:`cidr_limitations`                     		>= 4.11
-:ref:`encryption_ipsec` in tunneling mode   		>= 4.19
-:ref:`encryption_wg`                        		>= 5.6
-Socket-level LB                        		        >= 4.19.57, >= 5.1.16,  >= 5.2
-:ref:`kubeproxy-free`                       		>= 4.19.57, >= 5.1.16,  >= 5.2
-:ref:`bandwidth-manager`                    		>= 5.1
-:ref:`local-redirect-policy`				>= 4.19.57, >= 5.1.16,  >= 5.2
-Full support for :ref:`session-affinity`		>= 5.7
-:ref:`session-affinity` for kube-proxy ClusterIP	>= 4.10
-BPF-based proxy redirection				>= 5.7
-BPF-based host routing                      		>= 5.10
-Socket-level LB bypass in pod netns         		>= 5.7
-:ref:`egress-gateway`                       		>= 5.2
-VXLAN Tunnel Endpoint (VTEP) Integration    		>= 5.2
-IPv6 BIG TCP support                                    >= 5.19
-====================================================== 	===============================
+====================================================== ===============================
+Cilium Feature                                         Minimum Kernel Version
+====================================================== ===============================
+:ref:`encryption_ipsec` in tunneling mode              >= 4.19
+:ref:`encryption_wg`                                   >= 5.6
+Socket-level LB                                        >= 4.19.57, >= 5.1.16,  >= 5.2
+:ref:`kubeproxy-free`                                  >= 4.19.57, >= 5.1.16,  >= 5.2
+:ref:`bandwidth-manager`                               >= 5.1
+:ref:`local-redirect-policy`                           >= 4.19.57, >= 5.1.16,  >= 5.2
```

Comment:
Shouldn't we drop all the 4.19 mentions here?

Reply:
We _could_, what do you suggest we do with the 5.1 / 5.2 mentions then?

Or are you thinking that since kernel.org indicates that 5.1 and 5.2 are EOL, we can just stop mentioning these features entirely?

Reply:
I think that's actually quite reasonable, I'll fix that up.

Reply:
```
 ====================================================== ===============================
 Cilium Feature                                         Minimum Kernel Version
 ====================================================== ===============================
-:ref:`encryption_ipsec` in tunneling mode              >= 4.19
-:ref:`encryption_wg`                                   >= 5.6
-Socket-level LB                                        >= 4.19.57, >= 5.1.16,  >= 5.2
-:ref:`kubeproxy-free`                                  >= 4.19.57, >= 5.1.16,  >= 5.2
 :ref:`bandwidth-manager`                               >= 5.1
-:ref:`local-redirect-policy`                           >= 4.19.57, >= 5.1.16,  >= 5.2
+:ref:`egress-gateway`                                  >= 5.2
+VXLAN Tunnel Endpoint (VTEP) Integration               >= 5.2
+:ref:`encryption_wg`                                   >= 5.6
 Full support for :ref:`session-affinity`               >= 5.7
 BPF-based proxy redirection                            >= 5.7
-BPF-based host routing                                 >= 5.10
 Socket-level LB bypass in pod netns                    >= 5.7
-:ref:`egress-gateway`                                  >= 5.2
-VXLAN Tunnel Endpoint (VTEP) Integration               >= 5.2
+BPF-based host routing                                 >= 5.10
 IPv6 BIG TCP support                                   >= 5.19
 ====================================================== ===============================

```

Reply:
When I commented, I remember thinking this was exactly the time (i.e., too late) where I usually regret my reviews afterwards. And now, with some sleep, I see why you had kept it here. Regrets...

5.0 for example isn't supported for those features. And the only way to indicate that is to also mention 4.19.57. Dropping the lines entirely also works, but then that's not very consistent with the other 5.2 and 5.1 features in the table :confused:

Reply:
Yeah I'm going back and forth on this. 5.0 isn't supported either. But let's say we exclude 5.0,5.1,5.2 from consideration in this table because we aren't aware of any upstream kernel.org or distribution support. Then the 4.19.x + 5.1.y + 5.2.z cases become "yes it's always supported as long as you have the right 4.19.x version". For the other features that list 5.1, 5.2, etc., it helps to highlight that these are not supported on the 4.19.x kernel so you *will* need a newer kernel. You likely won't pick 5.1 or 5.2 or so on, you'll just pick a newer distribution that has a longer-term support kernel like 5.4 or even newer.

So I think that the latest state is probably reasonable?

Reply:
Yep, I'm convinced now :-) (that said, it's that time of the day again :grin:)

Thanks for thinking it through and writing it down :pray: